### ISSUE TITLE
the fixed dataloader bug (PCN)

### ISSUE THREAD

**Issue by beibeirory** [yuxumin/PoinTr#20]
You have fixed the bug in datasets/PCNDataset.py(#27), and updated the performance of PoinTr on PCN benchmark (CD from 8.38 to 7.26). But I can't find the fixed code in your code "PCNDataset.py", could you show me? Thanks in advance.

**Comment by yuxumin** [comment]
see [here](https://github.com/hzxie/GRNet/commit/8ccac42e6b556c83e4b641d79c5a7197d218933d#diff-e9f8eead35b36d20cde9cdbb850fbd1eb7e964cad6df019818a0a9551507e42e) and [here](https://github.com/yuxumin/PoinTr/commit/8f21f095877dfdd10a3849c2a8f95983faf6f507)

**Comment by beibeirory** [comment]
When I test the pretrained model on KITTI using "python KITTI_metric.py". It runs very  slowly (the result "36%, 846/2401" takes 12 hours ). Is it right?

**Comment by yuxumin** [comment]
Yes, i notice that, too.  Evaluating a sample using MMD metric requires comparisons with every sample in the PCNCars, taking a lot of time.

**Comment by beibeirory** [comment]
Thanks a lot. @yuxumin
